.env

```diff
@@ -1,6 +1,10 @@
 PORT = 4000
 
-API_URL = 'http://localhost:4000'
+API_URL = 'http://127.0.0.1:3000'
```

Comment:
I did this because "localhost" was not working, I don't think it should be implemented in master